pkg/controller/worker/machines_test.go

```diff
@@ -640,6 +641,48 @@ var _ = Describe("Machines", func() {
 					Expect(err).NotTo(HaveOccurred())
 					Expect(result).To(Equal(machineDeployments))
 				})
+
+				It("should delete the any old AWSMachineClasses", func() {
+					// mockAWSMachineClassList is the mocking object used to track the
+					// list of AWSMachineClass objects
+					mockAWSMachineClassList := machinev1alpha1.AWSMachineClassList{
```

Comment:
```suggestion
					awsMachineClassList := machinev1alpha1.AWSMachineClassList{
```

as it is not mock but rather test data.